### ISSUE TITLE
Not showing all available devices

### ISSUE THREAD

**Issue by IanAdd** [custom-cards/spotify-card#152]
**Describe the bug**

* [ y] Reproducable on a plain lovelace panel without card, style or theme mods
* [y ] Using latest version of spotcast and spotify-card [ ]
* [y ] Using latest stable version of Home Assistant
* [ y] I have Spotify Premium
* [y ] I have setup the Spotify integration in Home Assistant

**To Reproduce**

Mouse over the the device list and it shows 'Spotify Connect Devices' followed **only** by the name of my PC, then 'Chromecast Devices' and a full list of the google home/chromecast devices.

If I start the Spotify App on my PC and click to Connect to a device I get additional Spotify Connect devices showing - which are Sonos Speakers.

If I select on Spotify to play on one of these then the HA card shows the device's name as the playing device but the pop-up list still doesn't list it.

**Expected behavior**

If Spotify app thinks a device is a 'spotify connect' device then I would expect it to show in the card's list of devices.

**Screenshots**

**Environment (please complete the following information):**
Spotcast 3.5.10
lovelace card 2.3.8
Windows 10 PC using Chrome or Edge
HA 2021.6.2

**Configuration**
Add your configuration

**Additional context**

**Comment by fondberg** [comment]
Is this still an issue?

**Comment by IanAdd** [comment]
Yes. On my android phone, using HA app i have the same issue. Just lists chromecast devices, not Sonos speakers. But spotify app sees Sonos speakers as well as google cast devices. 
If i start spotify on my phone then use HA app it adds 'spotify connect' device of my phone 'android' to the list.

**Comment by fondberg** [comment]
Sonos are not supported as they are not available in the public Spotify api

**Comment by jwelvaert** [comment]
None of my Echo devices show up and 2 of my 3 Fire TV's show up. Any idea why this is the case?

**Comment by fondberg** [comment]
You have to debug the cast and zeroconf protocol. It is probably something with your home assistant and your router.
